CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [2.6.0] - TBD
+### Changed
+- Switch `IPriceFeedV2` to `IPriceFeedDispatcher` in `BaseToken` and `CollateralManager`.
+- Update `ClearingHouseConfig.setTwapInterval` to support 0 interval.
+
+## [2.5.0] - TBD
+### Added
+- Add `AccountBalance.getMarketRegistry()` to return the address of MarketRegistry contract.
+- Add `AccountBalance.getMarkPrice(address)` to return the mark price of given market.
+- Add `ClearingHouseConfig.getMarkPriceConfig()` to return marketTwapInterval and premiumInterval used for mark price calculations.
+- Add new event `MarketRegistryChanged` to `AccountBalance`, will emit in `setMarketRegistry(address)`
+
+### Changed
+- Move events in ClearingHouseConfig to IClearingHouseConfigEvent in IClearingHouseConfig.sol
+
+### Deprecated
+- BackstopLiquidityProvider from ClearingHouseConfig & IClearingHouseConfig and comments added to ClearingHouseConfigStorage
```

Comment:
The `backstopLiquidityProvider` is whitelist liquidators to liquidate the account has bad debt. It's the short-term solution to prevent the bad debt attack.

Now we already release new liquidator mechanism, the long-term solution of bad debt attack. Current liquidation is through taking over positions instead of market selling position.

Check the [internal docs](https://www.notion.so/perp/3799-4134-Bug-Reports-276fe67b8cb44578a7eda7bace70f77a#9b573a42c01f45b28289c05332132660 ) for details.